new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiConfigMap.java

```diff
@@ -0,0 +1,587 @@
+import static oracle.weblogic.kubernetes.actions.TestActions.dockerPush;
+import static oracle.weblogic.kubernetes.actions.TestActions.helmList;
+import static oracle.weblogic.kubernetes.actions.TestActions.installOperator;
+import static oracle.weblogic.kubernetes.actions.TestActions.uninstallOperator;
+import static oracle.weblogic.kubernetes.assertions.TestAssertions.dockerImageExists;
+import static oracle.weblogic.kubernetes.assertions.TestAssertions.domainExists;
+import static oracle.weblogic.kubernetes.assertions.TestAssertions.operatorIsRunning;
+import static oracle.weblogic.kubernetes.assertions.TestAssertions.podExists;
+import static oracle.weblogic.kubernetes.assertions.TestAssertions.podReady;
+import static oracle.weblogic.kubernetes.assertions.TestAssertions.serviceExists;
+import static oracle.weblogic.kubernetes.utils.ExecCommand.exec;
+import static oracle.weblogic.kubernetes.utils.FileUtils.checkDirectory;
+import static org.awaitility.Awaitility.with;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+
+// Test to create model in image domain and verify the domain started successfully
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+@DisplayName("Test to create model in image domain and start the domain")
+class ItMiiConfigMap implements LoggedTest {
+
+  // operator constants
+  private static final String OPERATOR_RELEASE_NAME = "weblogic-operator";
+  private static final String OPERATOR_CHART_DIR =
+      "../kubernetes/charts/weblogic-operator";
+  private static final String OPERATOR_IMAGE =
+      "oracle/weblogic-kubernetes-operator:3.0.0";
+  //"phx.ocir.io/weblogick8s/weblogic-kubernetes-operator:develop";
+
+  // mii constants
+  private static final String WDT_MODEL_FILE = "model1-wls.yaml";
+  private static final String MII_IMAGE_NAME = "mii-image";
+  private static final String APP_NAME = "sample-app";
+
+  // domain constants
+  private static final String DOMAIN_VERSION = "v7";
+  private static final String API_VERSION = "weblogic.oracle/" + DOMAIN_VERSION;
+
+  private static HelmParams opHelmParams = null;
+  private static V1ServiceAccount serviceAccount = null;
+  private static String opNamespace = null;
+  private static String domainNamespace = null;
+  private static ConditionFactory withStandardRetryPolicy = null;
+
+  private String domainUid = "domain1";
+  private String repoSecretName = "reposecret";
+  private String miiImage = null;
+  private String repoRegistry = "dummy";
+  private String repoUserName = "dummy";
+  private String repoPassword = "dummy";
+  private String repoEmail = "dummy";
+  private StringBuffer checkJdbc = null;
+
+  /**
+   * Install Operator.
+   */
+  @BeforeAll
+  public static void initAll() {
```

Comment:
Use "@Namespaces(1) List<String> namespaces" in method to get the namespaces for the test.

Reply:
done